### ISSUE TITLE
suggestions

### ISSUE THREAD

**Issue by madtomic** [tim-savage/SavageDeathSpawn#2]
/deathspawn list

Can this list be based on what the user had discovered? You can still let the OPs see all of the death spawns.

---

/deathspawn list

page 1 of 1 - this is always shown.

Is there a way to turn off this default message or change it or make it customizeable?

I would rather have custom name and follow by list of spawn discovered by player.

Discovered Towns or custom name.
- test
- test3

---

You can also try to use a flag system. You can list all the spawns but do something like this to let player know if they discovered a spawn or not.

Discovered Towns or custom name.
- test [not discover yet]
- test2
- test3 [not discover yet]

Spawns discovered, you can take the [not discover yet] out.

The way I envision and how to use this plugin. This is plugin can be used for discovery towns on my server. This plugin will replace the two other graveyards plugin that I had been testing.
I will probably set up command alias to replace /deathspawn with something like /towns

It would be nice if you can add in built in custom command alias.

Commands can be set in config.

# command alias for /deathspawn
command-alias: deathspawn

Again, thank you for making this awesome plugin!

**Comment by tim-savage** [comment]
Yes, I agree that the list command could use a little polishing. I originally intended it to be an admin only command, so I wasn't too concerned with the formatting or limiting it to only discovered locations. But letting players view the spawn names would be useful. I'll give it some thought and see what I can come up with for customizations, while keeping the configuration as simple as possible.

Adding a permission node deathspawn.list.undiscovered is one possibility. Then the deathspawn.admin node can include that as a child node, and certain players or groups can also be granted that permission if desired.

Allowing a custom command alias to be set in the config is probably possible, but the way the bukkit command api is designed doesn't really lend itself to this. Creating a command alias in the server's command.yml file is the preferred method. The plugin does have an official, but so far undocumented, registered alias of /sds.

**Comment by tim-savage** [comment]
The "Page 1 of 1" header will inform the user if there are more than "page" of listings. The DeathSpawns per page is currently hard coded to 5, so if you create more than 5 DeathSpawns, it will show "Page 1 of 2" etc.

You can show subsequent pages by putting a page number after the list command. I forgot to add that to the documentation, so I'll update the docs accordingly.

**Comment by madtomic** [comment]
Ok, thank you for getting back with me.
